internal/rbd/clone.go

```diff
@@ -177,6 +177,13 @@ func (rv *rbdVolume) createCloneFromImage(ctx context.Context, parentVol *rbdVol
 	}
 
 	defer func() {
+		if err != nil || errClone != nil {
+			cErr := cleanUpSnapshot(ctx, tempClone, cloneSnap, rv, rv.conn.Creds)
+			if cErr != nil {
+				klog.Errorf(util.Log(ctx, "failed to cleanup image %s or snapshot %s: %v"), cloneSnap, tempClone, cErr)
```

Comment:
you're touching this code, so please use the `util.*Log()` function too.

Reply:
sent PR for using util logger here https://github.com/ceph/ceph-csi/pull/1409